source/settings.txt

```diff
@@ -1,5 +1,114 @@
 .. _vsce-settings:
 
-============================
-MongoDB for VS Code Settings
-============================
+===============
+|vsce| Settings
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Configure Settings
+------------------
+
+To configure |vsce| settings:
+
+.. include:: /includes/steps/configure-settings.rst
+
+Settings
+--------
+
+The following table lists the available settings for 
+|vsce|:
+
+.. list-table::
+   :widths: 25 50 25
+   :header-rows: 1
+   
+   * - Setting
+     - Description
+     - Default
+
+   * - :guilabel:`Shell`
+     - MongoDB shell to use. Choose either:
+       
+       - ``mongo`` (the legacy MongoDB shell), or
+
+       - ``mongosh`` (the new MongoDB shell)
+
+     - ``mongo``
+
+   * - :guilabel:`Show`
+     - If enabled, shows the MongoDB view in the Visual Studio Code 
+       left navigation panel.
+     - Enabled
+
+   * - :guilabel:`Max Number of Problems`
+     - The maximum number of problems |vsce| will generate 
+       in the Visual Studio Code :guilabel:`Problems` view.
+     - ``100``
+
+   * - :guilabel:`Trace: Server`
+     - MongoDB language server logging settings. 
+     
+       Edit this setting through the 
+       ``mongodbLanguageServer.trace.server`` property in your project's
+       ``settings.json`` file. 
+
+       Within the ``mongodbLanguageServer.trace.server`` object, you 
+       can specify:
+
+       - ``format``, and
+
+       - ``verbosity``
+       
+       Output logs are viewable by selecting 
+       :guilabel:`MongoDB Language Server` from the dropdown in the 
+       :guilabel:`Output` view, or with the 
+       `Language Server Protocol Inspector <https://github.com/Microsoft/language-server-protocol-inspector>`__.
+
+     - .. code-block:: json
+     
+          {
+            "format": "json",
+            "verbosity": "verbose"
+          }
+
+   * - :guilabel:`Confirm Run All`
+     - If enabled, shows a confirmation message before running 
+       commands in a playground.
+     - Enabled
+
+   * - :guilabel:`Default Connection Saving Location`
+     - Sets if and where new connections are saved.
```

Comment:
I think we should explain what the options are. I was including that information as part of the Connect page updates, but it'd be helpful to include here as well. Here is what I had, if you just want to include it in your updates:

```
.. list-table::
   :header-rows: 1
   :widths: 20 40

   * - Setting
     - Description

   * - ``Global``
     - Save your connection globally in VS Code, so it can be accessed
       from any workspace.

   * - ``Workspace``
     - Save your connection in your workspace. You cannot
       access the connection from a different workspace.

   * - ``Session``
     - Save the connection for only this VS Code session. The connection
       is lost when you close VS Code.
```

Suggest putting this in an include for single-sourcing.

Reply:
Will do, adding a file to /includes.